newIDE/app/src/PropertiesEditor/MeasurementUnitDocumentation.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import { Column } from '../UI/Grid';
+import Text from '../UI/Text';
+
+// Strings are used instead of gdMeasurementUnit
+// because the C++ instance would be dead when the component is rendered.
+type Props = {|
+  label: string,
+  description: string,
+  elementsWithWords: string,
+|};
```

Comment:
Is it ok?

Reply:
I'm not sure what "dead" means

Reply:
The `gdMeasurementUnit` instances are only living during their iteration on the properties when building the property editor view. As the rendering of `MeasurementUnitDocumentation` is done outside of this loop, the memory referenced by the `gdMeasurementUnit` instance is reused and it display wrong values.